src/Response.php

```diff
@@ -82,6 +95,46 @@ public function getBody()
         return $this->body;
     }
 
+    /**
+     * @return array
+     */
+    public function getRecords()
+    {
+        return $this->records;
+    }
+
+    /**
+     * @return array
+     */
+    public function getRawResponse()
+    {
+        return $this->response;
+    }
+
+    /**
+     * @return string
+     */
+    public function getScriptResult()
+    {
+        if (isset($this->response['scriptResult']) || array_key_exists('scriptResult', $this->response)){
+            return $this->response['scriptResult'];
+        } else {
+            return;
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function getScriptError()
+    {
+        if (isset($this->response['scriptError']) || array_key_exists('scriptError', $this->response)){
+            return $this->response['scriptError'];
+        } else {
+            return;
```

Comment:
Same as above, This needs to do something else than just "return;". As of PHP 7.1, that is a compile-time error.

https://www.php.net/manual/en/function.return.php

Reply:
@JacobTaylor-RCC I updated this code so it will return an empty string if the script error is empty. Unsure if returning null might be more appropriate. Open to your input there.